Leanplum-SDK/Classes/Managers/Networking/LPRequestFactory.m

```diff
@@ -0,0 +1,19 @@
+
+#import "LPRequestFactory.h"
+#import "LeanplumRequest.h"
+
+@implementation LPRequestFactory
+
++ (id<LPRequesting>)get:(NSString *)apiMethod params:(NSDictionary *)params {
+//    LPLogType level = [apiMethod isEqualToString:LP_METHOD_LOG] ? LPDebug : LPVerbose;
+//    LPLog(level, @"Will call API method %@ with arguments %@", apiMethod, params);
+    return [[LeanplumRequest alloc] initWithHttpMethod:@"GET" apiMethod:apiMethod params:params];
+}
+
++ (id<LPRequesting>)post:(NSString *)apiMethod params:(NSDictionary *)params {
```

Comment:
This API is a little strange. `LPRequestFactory.get` intuitively sounds like it's getting a request. Not creating a `GET` request. Maybe something like `makeGet` and `makePost` https://en.wikipedia.org/wiki/Factory_method_pattern